open-company/open-company-web
# Drafts fixes

Card: https://trello.com/c/ej5rN2aS

To test:
- go to All Posts
- [x] do you see WRT count? Good
- [x] do you see the more menu on desktop (on hover) and mobile (always)? Good
- go to a section
- [x] do you see WRT count? Good
- [x] do you see the more menu on desktop (on hover) and mobile (always)? Good
- go to Drafts
- [x] do you NOT see WRT count? Good
- [x] do you NOT see the more menu on desktop (on hover) and mobile (always)? Good
- [x] do you see the continue editing and delete draft? Good
